core/src/test/java/org/infinispan/configuration/module/MyParserExtension.java

```diff
@@ -12,12 +13,23 @@
  * MyParserExtension. This is a simple extension parser which parses modules in the "urn:infinispan:config:mymodule" namespace
  *
  * @author Tristan Tarrant
+ * @author Réda Housni Alaoui
  * @since 5.2
  */
 @Namespace(uri = "urn:infinispan:config:mymodule", root = "sample-element")
 @Namespace(root = "sample-element")
 public class MyParserExtension implements ConfigurationParser {
 
+   private static final AtomicReference<RuntimeException> nextGetNamespacesException = new AtomicReference<>();
```

Comment:
This is not a recommended way of forcing an exception in production code. You should be able to use something like: https://github.com/infinispan/infinispan/blob/main/core/src/test/java/org/infinispan/manager/CacheManagerTest.java#L420